### ISSUE TITLE
404s on gcc4.4 portability tests

### ISSUE THREAD

**Issue by jcanizales** [grpc/grpc#5821]
It only happens on the gcc4.4 tests, and it happens every so often:

### apt-get update

https://grpc-testing.appspot.com/job/gRPC_portability_master/954/language=c,scenario=linux_x64_gcc4.4/console
https://grpc-testing.appspot.com/job/gRPC_portability_master/961/language=c,scenario=linux_x64_gcc4.4/console

```
Get:5 http://httpredir.debian.org squeeze-lts Release [13.0 kB]
Err http://httpredir.debian.org squeeze/main amd64 Packages
  404  Not Found [IP: 198.145.20.143 80]
Err http://httpredir.debian.org squeeze-updates/main amd64 Packages
  404  Not Found [IP: 198.145.20.143 80]
Fetched 556 kB in 1s (368 kB/s)
W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze-lts/Release  Unable to find expected entry  main/binary-amd64/Packages in Meta-index file (malformed Release file?)

W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze/main/binary-amd64/Packages.gz  404  Not Found [IP: 198.145.20.143 80]

W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze-updates/main/binary-amd64/Packages.gz  404  Not Found [IP: 198.145.20.143 80]

E: Some index files failed to download, they have been ignored, or old ones used instead.
```

https://grpc-testing.appspot.com/job/gRPC_portability_master/963/language=c,scenario=linux_x64_gcc4.4/console

```
Get:4 http://httpredir.debian.org squeeze-lts Release [13.0 kB]
Err http://httpredir.debian.org squeeze/main amd64 Packages
  404  Not Found [IP: 128.31.0.66 80]
Get:5 http://security.debian.org squeeze/updates/main amd64 Packages [455 kB]
Err http://httpredir.debian.org squeeze-updates/main amd64 Packages
  404  Not Found [IP: 128.31.0.66 80]
Fetched 556 kB in 0s (891 kB/s)
W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze-lts/Release  Unable to find expected entry  main/binary-amd64/Packages in Meta-index file (malformed Release file?)

W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze/main/binary-amd64/Packages.gz  404  Not Found [IP: 128.31.0.66 80]

W: Failed to fetch http://httpredir.debian.org/debian/dists/squeeze-updates/main/binary-amd64/Packages.gz  404  Not Found [IP: 128.31.0.66 80]

E: Some index files failed to download, they have been ignored, or old ones used instead.
```

### wget

https://grpc-testing.appspot.com/job/gRPC_portability_master/962/language=c,scenario=linux_x64_gcc4.4/console

```
Step 7 : RUN wget http://openssl.org/source/openssl-1.0.2f.tar.gz
 ---> Running in 201f23045da2
--2016-03-18 01:03:21--  http://openssl.org/source/openssl-1.0.2f.tar.gz
Resolving openssl.org... 194.97.150.234, 2001:608:c00:180::1:ea
Connecting to openssl.org|194.97.150.234|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2016-03-18 01:03:21 ERROR 404: Not Found.
```

**Comment by jtattermusch** [comment]
Interesting, it looks like that building the docker image `tools/dockerfile/test/cxx_squeeze_x64` is broken. 
It only passes on the workers where the image has been build previously and the cached version is used.

**Comment by soltanmm** [comment]
Seen again:
[1](https://grpc-testing.appspot.com/job/gRPC_portability_master/981/language=c,scenario=linux_x64_gcc4.4/console) [2](https://grpc-testing.appspot.com/job/gRPC_portability_master/969/language=c,scenario=linux_x64_gcc4.4/console) [3](https://grpc-testing.appspot.com/job/gRPC_portability_master/979/language=c,scenario=linux_x64_gcc4.4/console)

**Comment by jtattermusch** [comment]
Seems fixed https://grpc-testing.appspot.com/job/gRPC_portability_master/986/

**Comment by jcanizales** [comment]
Happening again: https://grpc-testing.appspot.com/job/gRPC_portability_pull_requests/1983/language=c,scenario=linux_x64_gcc4.4_bo/console

**Comment by jtattermusch** [comment]
Not happening again, your PR is against 0.13 branch that doesn't have the fix #5829. I don't think it's worth backporting this fix.
